addon_service/oauth1/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+from addon_service.common.base_model import AddonsServiceBaseModel
+
+
+class OAuth1ClientConfig(AddonsServiceBaseModel):
```

Comment:
`auth_url` is `auth_callback_url`, I'll change it if it's confusing

Reply:
For the record: we're storing the `auth_callback_url` explicitly in case there are services that won't allow us to update the list of callbacks without invalidating our client and/or old credentials. That way, if necessary, we can encode this as `https://[environment.]osf.io/oauth/callback/[provider]` -- where we already have things set up to forward requests to GV if appropriate.